### DISCUSSION TITLE
v0.17.5

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#2995] [Versions & Releases]
## What's Changed
* Fix fsspec  requirement by @WeberJulian in https://github.com/coqui-ai/TTS/pull/2970
* Add coqui blog post by @osanseviero in https://github.com/coqui-ai/TTS/pull/2949
* fix: xtts not taking into account device flag by @loupzeur in https://github.com/coqui-ai/TTS/pull/2951
* fix package versions by @WeberJulian in https://github.com/coqui-ai/TTS/pull/2990

## New Contributors
* @osanseviero made their first contribution in https://github.com/coqui-ai/TTS/pull/2949
* @loupzeur made their first contribution in https://github.com/coqui-ai/TTS/pull/2951

**Full Changelog**: https://github.com/coqui-ai/TTS/compare/v0.17.4...v0.17.5

<hr /><em>This discussion was created from the release <a href='https://github.com/coqui-ai/TTS/releases/tag/v0.17.5'>v0.17.5</a>.</em>

**Comment by hau99** [comment]
Nice
